tests/conftest.py

```diff
@@ -32,6 +32,9 @@
 # a dynamic number of ports. GitHub Issue: Azure/sonic-swss#1384.
 NUM_PORTS = 32
 
+# FIXME: Voq asics will have 16 fabric ports created (defined in Azure/sonic-buildimage#6185).
+# Right now, we set FABRIC_NUM_PORTS to 0, and change to 16 when PR#6185 merges.
+FABRIC_NUM_PORTS = 0
```

Comment:
it looks like https://github.com/Azure/sonic-buildimage/pull/6185 has been merged. can you check to 16?

Reply:
No, 6158 merged accidentally, but then it got reverted.